docs/event-bus/docs/03-02-event-flow-requirements.md

```diff
@@ -3,39 +3,31 @@ title: Event flow requirements
 type: Details
 ---
 
-The Event Bus enables a successful flow of the Events in Kyma when:
+The Event Bus enables a successful flow of Events in Kyma when:
 
-- The [EventActivation](#details-event-flow-requirements-activate-events) is in place.
-- You create a [Subscription](#details-event-flow-requirements-consume-events) Kubernetes custom resource and register the webhook for the lambda or a service to consume the Events.
+- You have enabled the [EventActivation](docs/components/event-bus#concepts) controller.
+- You have created a [Subscription](/docs/components/event-bus#custom-resource-subscription) custom resource and registered the webhook for a lambda or service to consume the Events.
 - The Events are [published](#details-event-flow-requirements-event-publishing).
 
 ## Details
 
-See the following subsections for details on each requirement.
+Read the following subsections for details on the requirements.
 
 ### Activate Events
 
-To receive Events, use EventActivation between the Namespace and the Application (App).
+Use the EventActivation to ensure the Event flow between the Namespace and the Application (App). You can also simply [bind](/docs/components/application-connector#getting-started-bind-an-application-to-a-namespace) the App to the Namespace.
```

Comment:
and after that add to the namespace from the service catalog. I see that is a separate step.